### PR TITLE
Fix bugs with label 'target 2.0.1'

### LINKED ISSUE #11546: The status “Succeed” should say “Succeeded”
/api/v2.0/replication/executions

image

### LINKED ISSUE #11652: Helm Hub replication enabled even with ChartMuseum disabled
If you are reporting a problem, please make sure the following information are provided:

**Expected behavior and actual behavior:**
Helm hub replication to finish

**Steps to reproduce the problem:**
Create a helm hub registry with spec:
```json
{
  "credential": {
    "access_key":"",
    "access_secret":"",
    "type":"basic"
  },
  "description":"test",
  "insecure":false,
  "name":"helm_hub",
  "type":"helm-hub",
  "url":"https://hub.helm.sh/"
}
```

Create any pull-based replication rule from that repository. 

**Versions:**
Please specify the versions of following systems.

- harbor version: [v1.10.1]
- docker engine version: [18.04]

**Additional context:**

Logs from jobservice:

```
2020-04-17T15:55:19Z [ERROR] [/jobservice/runner/redis.go:89]: Job 'REPLICATION:72bd1ac982960b13af3e89d9' exit with error: run error: http error: code 404, message <!DOCTYPE HTML$
<html>
<head>
<meta charset="utf-8">
<title>404 - Page Not Found</title>
<style>

*{
        maring: 0;
        padding: 0;
}

a:link{
        color: #1f3759;
        text-decoration: none;
}
a:active{
        color: #1f3759;
        text-decoration: none;
}
a:hover{
        color: #9fb7d9;
        text-decoration: none;
}
a:visited{
        color: #1f3759;
        text-decoration: none;
}

a.underline, .underline{
        text-decoration: underline;
}

#content{
        margin: 0 auto;
        width: 800px;
}

#main-body{
        text-align: center;
}

.status-code {
    font-weight: bolder;
    font-size: 4em;
    vertical-align: middle;
}

.status-text {
    font-weight: bold;
    font-size: 3em;
    margin-left: 10px;
    vertical-align: middle;
}

.status-subtitle {
    font-size: 18px;
}

</style>
</head>
<body>
    <div id="content">
        <div id="main-body">
                        <div>
                        <span class="status-code">404</span>
                        <span class="status-text">Page Not Found</span>
                    </div>
                    <div class="status-subtitle">
                        <p><a href="/harbor" class="underline">Home</a></p>
                    </div>
        </div>
    </div>
</body>
</html>
```

### LINKED ISSUE #11993: Summary of chart version in dark mode not all dark
## Expected
Display all gui elements in Summary Chart Versions in dark background in dark mode

## Actual
Some gui elements in Summary Chart Versions have white background in dark mode

## Steps to reproduce the problem
- have a chart avaliable in Project/charts
- select dark mode 
- Go to view details of chart version in portal

## Versions 

- harbor version: [2.0.0]
- docker engine version: [19.03.8]
- docker-compose version: [1.25.5]
    
## Additional context:

tested with following browsers
- Chrome 83.0.4103.61 (Official Build) (64-bit) 
- Firefox 76.0.1 (64-bit)

screenshot harbor summary chart version dark mode

### LINKED ISSUE #12047: land on the ‘Repositories’ page instead of the ‘Summary’ page
I would like to land on the ‘Repositories’ page instead of the ‘Summary’ page when drilling down to any project from the ‘Projects’ page on Harbor web console.  Most of the time I’m just walking the directory to access my image in various repositories under various projects. Clicking ‘Repositories’ each time is unnecessary and tiresome

Screen Shot 2020-05-23 at 9 33 02 AM

### LINKED ISSUE #12048: add last modified timestamp column to 'Repositories' page
I want to add a ‘last modified’ timestamp column to the ‘Repositories’ page on Harbor web console, this should display the last modified time of any artifact within this repository.  Or maybe show ‘last pulled time’ and ‘last pushed time’ separately

Screen Shot 2020-05-23 at 9 35 12 AM

### LINKED ISSUE #12049: UI bug when contracting window
Screen Shot 2020-05-14 at 5 34 07 AM

### LINKED ISSUE #12050: in repo view show the project name
Within repository view, I'd like to be able tell which project I'm in.  

Screen Shot 2020-05-23 at 5 27 17 AM

Screen Shot 2020-05-23 at 5 29 58 AM

### LINKED ISSUE #12051: timestamp format inconsistency
Screen Shot 2020-05-23 at 8 15 17 AM

### LINKED ISSUE #12052: scanned items should be unchecked
Screen Shot 2020-05-23 at 8 42 25 AM

### LINKED ISSUE #12053: adding tag retention UI issue
insert linebreak before 'enter multiple comma separate..' at the very below, right now it's too close to the tag specification box

Screen Shot 2020-05-23 at 9 44 11 AM

### PR DESCRIPTION
**Pull request by AllForNothing**
Temp Env :  https://10.185.12.35:4200/

Fix #12053 
Fix #12052 
Fix #12050 
Fix #12049 
Fix #12048 
Fix #12047 
Fix #11652 
Fix #11546 
Fix #12051 
Fix #11993
Signed-off-by: AllForNothing <sshijun@vmware.com>

### CODE REVIEW

File: src/portal/src/app/project/repository/artifact/artifact-additions/artifact-vulnerabilities/artifact-vulnerabilities.component.html [hunk 1]
```diff
@@ -10,10 +10,10 @@
         <clr-datagrid [clrDgLoading]="loading">
             <clr-dg-action-bar>
                 <div class="clr-row center">
-                    <div class="clr-col-1">
+                    <div class="ml-05">
                         <button (click)="scanNow()" type="button"  class="btn  btn-secondary" [clrLoading]="scanBtnState" [disabled]="!(hasEnabledScanner && hasScanningPermission && !onSendingScanCommand)"><clr-icon shape="shield-check" size="16"></clr-icon>&nbsp;{{'VULNERABILITY.SCAN_NOW' | translate}}</button>
                     </div>
-                    <div class="clr-col">
+                    <div class="ml-1">
```
Comment (selina-cui) [inline src/portal/src/app/project/repository/artifact/artifact-additions/artifact-vulnerabilities/artifact-vulnerabilities.component.html:16]: ml-1 is not defined in artifact-vulnerabilities.component.scss

File: src/portal/src/lib/components/replication/replication.component.ts [hunk 2]
```diff
@@ -82,6 +82,10 @@ export class SearchOption {
   pageSize: number = DEFAULT_PAGE_SIZE;
 }
 
+const STATUS_MAP = {
+  "Succeed": "Succeeded"
```
Comment (selina-cui) [inline src/portal/src/lib/components/replication/replication.component.ts:86]: Could you put the constant STATUS_MAP in some golable variable file or language json file?  We needn't define it twice in this file and replication-tasks.component.ts.

File: src/portal/src/app/project/repository/artifact/artifact-additions/artifact-vulnerabilities/artifact-vulnerabilities.component.html (hunk 1 above)
Comment (AllForNothing) [reply src/portal/src/app/project/repository/artifact/artifact-additions/artifact-vulnerabilities/artifact-vulnerabilities.component.html:16]: For widely used css,  them are in global css

File: src/portal/src/i18n/lang/zh-tw-lang.json [hunk 3]
```diff
@@ -700,7 +700,8 @@
     "DEPLOY": "部署",
     "ADDITIONAL_INFO": "添加信息",
     "REPO_NAME": "鏡像倉庫",
-    "MARKDOWN": "支持使用Markdown進行樣式設置"
+    "MARKDOWN": "支持使用Markdown進行樣式設置",
+    "LAST_MODIFIED": "Last Modified Time"
```
[the remark this answers is not in the document]
Comment (AllForNothing) [reply src/portal/src/i18n/lang/zh-tw-lang.json:704]: need't. We only cover English and simple Chinese

File: src/portal/src/lib/components/replication/replication.component.ts (hunk 2 above)
Comment (AllForNothing) [reply src/portal/src/lib/components/replication/replication.component.ts:86]: In fact , it's a back-end api bug.  So modify where it appears

**Pull request merged**